src/content/docs/logs/logs-context/custom-tags-agent-forwarder-logs.mdx

```diff
@@ -7,11 +7,6 @@ tags:
   - Tag
   - Labels
 metaDescription: For apps monitored by New relic language agents, learn how to add custome tags (labels) to logs.
-redirects:
-  - /docs/enable-logs-context
-  - /docs/apm/agents/net-agent/configuration/net-agent-configuration
-  - /docs.newrelic.com/docs/apm/agents/ruby-agent/configuration/ruby-agent-configuration/
-  - /docs/logs/new-relic-logs/enable-logs-context/enable-logs-context-go
```

Comment:
These redirects were trying to redirect a bunch of other pages to this one for some reason, my guess is this was a copy paste error that then got propagated.